docs/BUILDS.md

```diff
@@ -15,7 +15,7 @@ Note: `minimal` variant is not listed as it shouldn't be used outside of the [up
 | USE_DOMOTICZ              | - | x / x | x | x | x | - |
 | USE_HOME_ASSISTANT        | - | - / - | - | - | - | - |
 | USE_TASMOTA_DISCOVERY     | x | x / x | x | x | x | x |
-| USE_MQTT_TLS*             | - | - / x | - | - | - | - |
+| USE_MQTT_TLS              | - | - / x | - | - | - | - | USE_MQTT_TLS is enabled by default in every ESP32 variants
```

Comment:
Please remove any changes from BUILDS.md

BUILDS.md and I2CDEVICES.md are actually automatically copied from Tasmota repo and should not be edited in the docs repo
To update them, please fill a PR on arendst/Tasmota

Thanks

Reply:
Got it, reverted `BUILDS` and `I2CDEVICES`. Since I added links from a couple other locations I probably will just leave those as-is, but good to know where they come from.